content/foundation/governing-board/committees.md

```diff
@@ -0,0 +1,167 @@
++++
+title = "Governing Board Committees"
+weight = 3
+extra.summary = """
+The Committees of the Governing Board are subgroups focusing on specific areas of interest to The Matrix Foundation.
+"""
++++
+
+The Committees of the Governing Board are subgroups focusing on specific areas of interest to The Matrix Foundation.
+They are the primary working units of the Board and consist exclusively of Board members.
+Committees host [Working Groups](@/foundation/working-groups/index.md) to enable broader community participation in the Foundation.
```

Comment:
I think it could be made clearer that working group members are not necessarily part of the governing board. How about the following?

```suggestion
People who are not part of the Governing Board can also give their time to help the Foundation. Such volunteers can join [Working Groups](@/foundation/working-groups/index.md) that contain at least a Governing Board Member interested in their topic, and volunteers.
```

Reply:
I agree with both of you. How about this?
---
Committees contain multiple [Working Groups] to enable broader community participation in the Foundation. Working Groups are made of volunteer members, along with a Board Member to co-ordinate with the rest of the Board & other Groups.
---
I think that hits the notes?